docs/setup/keadm.md

```diff
@@ -96,70 +96,229 @@ KubeEdge edgecore is running, For logs visit:  /var/log/kubeedge/edgecore.log
+    | d40ad4007d6f9 | k8s_POD_weave-net-fr9vx_kube-system_...         |
+    | f59bdacbfc48  | k8s_kube-proxy_kube-proxy-s8476_kube-system_... |
+    | f15a4b9aae7a  | k8s_POD_kube-proxy-s8476_kube-system_.....      |
+
+
+    Kill those `Network` related containers to make restart 
+    ``` shell
+    docker container kill d40ad407d6f9
+    docker container kill f59bdacbfc48
+    docker container kill f15a4b9aae7a
+    systemctl restart edgecore.service
+    ```
+
 ### Support Metrics-server in Cloud
 1. The realization of this function point reuses cloudstream and edgestream modules. So you also need to perform all steps of *Enable `kubectl logs` Feature*.
 
-2. Since the kubelet ports of edge nodes and cloud nodes are not the same, the current release version of metrics-server(0.3.x) does not support automatic port identification, so you need to manually compile the image yourself now.
+2. Since the kubelet ports of edge nodes and cloud nodes are not the same, the current release version of metrics-server(0.3.x) does not support automatic port identification (It is the 0.4.0 feature), so you need to manually compile the image from master branch yourself now.
 
-```bash
-git clone https://github.com/kubernetes-sigs/metrics-server.git
-cd metrics-server
-make container
-docker images
-docker tag 2d9eb6e0d887 metrics-server-kubeedge:latest
-```
+    Git clone latest metrics server repository:
+
+    ```bash
+    git clone https://github.com/kubernetes-sigs/metrics-server.git
+    ```
+   
+    Go to the metrics server directory:
+
+    ```bash
+    cd metrics-server
+    ```
+
+    Make the docker image:
+
+    ```bash
+    make container
+    ```
+
+    Check if you have this docker image:
+
+    ```bash
+    docker images
+    ```
+
+    |                  REPOSITORY                           |                    TAG                   |   IMAGE ID   |     CREATE     |  SIZE  |
+    |-------------------------------------------------------|------------------------------------------|--------------|----------------|--------|
+    | gcr.io/k8s-staging-metrics-serer/ metrics-serer-amd64 | 6d92704c5a68cd29a7a81bce68e6c2230c7a6912 | a24f71249d69 | 19 seconds ago | 57.2MB |
+    | metrics-server-kubeedge                               |                 latest                   | aef0fa7a834c | 28 seconds ago | 57.2MB |
+
+
+    Make sure you change the tag of image by using its IMAGE ID to be compactable with image name in yaml file.
+
+    ```bash
+    docker tag a24f71249d69 metrics-server-kubeedge:latest
+    ```
 
 3. Apply the deployment yaml. For specific deployment documents, you can refer to https://github.com/kubernetes-sigs/metrics-server/tree/master/manifests.
 
+    Before you deploy metrics-server, you have to make sure that you deploy it on the node which has apiserver deployed on. In this case, that is the master node. As a consequence, it is needed to make master node schedulable by the following command:
```

Comment:
The request to retrive metric-data from edgecore:10250 through tunnel between cloudcore and edgecore is issued by metric-server. To direct this req to cloudcore tunnel/stream server, we need iptables, so the importance is ** those iptables above must be applyed on the machine metric-server runs on**.
```
 iptables -t nat -A OUTPUT -p tcp --dport 10350 -j DNAT --to $CLOUDCOREIPS:10003
```
Of course, deploy ms on master is a good solution, because master has been applied those iptables for enabling logs feature , I think it is better to note the key point there as well.